examples/nextjs-auth-demo/package.json

```
@@ -13,8 +13,8 @@
   "dependencies": {
     "@auth0/nextjs-oidc": "file:../..",
     "delay": "^4.3.0",
+    "isomorphic-unfetch": "^3.0.0",
     "next": "^9.0.4",
```

```suggestion
    "next": "^9.0.5",
```
Could bump this `v9.0.5` now